csrc/device_lower/analysis/predicate_elimination.cpp

```diff
@@ -25,6 +25,16 @@ namespace nvfuser {
 
 namespace {
 
+TensorView* getTensorView(Val* v) {
```

Comment:
Add support for `kir::TensorIndex` to `assertOnWarpOps`

Reply:
We already have such thing:
https://github.com/NVIDIA/Fuser/blob/34dd43a6b88c07d000647ac51e6c9d04a1e2276f/csrc/device_lower/utils.cpp#L271

https://github.com/NVIDIA/Fuser/blob/34dd43a6b88c07d000647ac51e6c9d04a1e2276f/csrc/ir/utils.h#L784-L788